client/src/components/Footer.js

```diff
@@ -15,28 +15,42 @@ const Footer = styled.footer`
 `;
 
 const List = styled.ul`
+  width: 100%;
   display: flex;
-  felx-direction: row;
+  flex-direction: row;
   justify-content: space-between;
 
   @media (max-width: 900px) {
     flex-direction: column;
+    justify-content: space-evenly; /* I tried to use this to create some space when
+    screen size is small but it didn't work..*/
   };
 `;
 
 const ListItem = styled.li`
+  width: 20%;
   display: flex;
-  felx-direction: row;
+  flex-direction: row;
+  justify-content: space-around;
   color: ${props => props.theme.textColor1};
-  
-  @media (min-width: 768px) {
+  @media only screen and (max-width: 995px) {
```

Comment:
lets get the resolution width from a variable in the theme

let have variables such as `small-screen`, `medium-screen`, `big-screen`